clickhouse/values.yaml

```diff
@@ -375,6 +375,9 @@ clickhouse:
       max_part_loading_threads: auto
       # If the number of broken parts in a single partition exceeds the max_suspicious_broken_parts value, automatic deletion is denied.
       max_suspicious_broken_parts: 100
+    ##
+    ## Allows to override default Clickhouse config via override.xml
+    configOverride: ""
```

Comment:
Can you maybe add your example here (commented) ?

Reply:
done :)